draft-ietf-quic-qpack.md

```diff
@@ -627,14 +632,14 @@ type HTTP_WRONG_STREAM_COUNT. These streams MUST NOT be closed. Closure of
 either unidirectional stream type MUST be treated as a connection error of type
 HTTP_CLOSED_CRITICAL_STREAM.
 
-An endpoint MAY avoid creating its own encoder stream if it's not going to be
```

Comment:
I do not have the impression that HTTP/3 defines ownership of unidirectional streams, but "create" is unambiguous anyway.